src/Init/Prelude.lean

```diff
@@ -1731,19 +1715,22 @@ def Nat.ble : @& Nat → @& Nat → Bool
   | succ n, succ m => ble n m
 
 /--
-An inductive definition of the less-equal relation on natural numbers,
-characterized as the least relation `≤` such that `n ≤ n` and `n ≤ m → n ≤ m + 1`.
+Non-strict inequality of natural numbers, usually accessed via the `≤` operator.
```

Comment:
```suggestion
Non-strict, or weak inequality of natural numbers, usually accessed via the `≤` operator.
```

Reply:
I'll add it with an extra comma after "weak"